app/assets/javascripts/discourse/app/components/modal/reject-reason-reviewable.hbs

```diff
@@ -0,0 +1,26 @@
+<DModal
+  @bodyClass="explain-reviewable"
```

Comment:
Technically this should be `reject-reason-reviewable-modal__explain-reviewable` because of BEM rules